Phaethon/InternalApi/Controllers/ElementController.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
```

Comment:
You should remove the unused references

Reply:
i could, but it doesnt realy change enything

Reply:
Would make the code cleaner.

Reply:
would it realy, sure it would reduce ammount of text in file, but i never realy look there, unless i want to add reference, and than if its not clean enough i remove unused stuff

Reply:
tbh i think having 2 web apis is way bigger issue, but we will talk about it in meeting